tests/functional/conftest.py

```diff
@@ -1,100 +1,344 @@
 from tests.scripts.pds_request import GenericPdsRequestor, PdsRecord
 import pytest
-
+import os
+from pytest_bdd import given, when, then, parsers
+from jsonpath_rw import parse
+from pytest_check import check
+import urllib
 from .utils.apigee_api_apps import ApigeeApiDeveloperApps
 from .utils.apigee_api_products import ApigeeApiProducts
-from .config_files import config
+from .configuration import config
+from .configuration.config import BASE_URL, PDS_BASE_PATH
 import random
+from requests import Response, get, patch
 import json
 from typing import Union
-
+from .data.searches import Search
+from .data.updates import Update
+from .data import searches
+from .data import updates
+from .data import patients
+from .data.patients import Patient
+from copy import copy
 
 from pytest_nhsd_apim.identity_service import (
     ClientCredentialsConfig,
     ClientCredentialsAuthenticator,
 )
 
 from pytest_nhsd_apim.apigee_apis import (
-    ApigeeClient,
     ApiProductsAPI,
-    ApigeeNonProdCredentials,
     DeveloperAppsAPI,
 )
 
-import logging
-
-LOGGER = logging.getLogger(__name__)
-DEVELOPER_EMAIL = "apm-testing-internal-dev@nhs.net"
-
-
-@pytest.fixture()
-def client():
-    config = ApigeeNonProdCredentials()
-    return ApigeeClient(config=config)
+FILE_DIR = os.path.dirname(__file__)
+RESPONSES_DIR = os.path.join(FILE_DIR, 'data', 'responses')
 
 
 @pytest.fixture()
-def api_products(client):
-    return ApiProductsAPI(client=client)
+def developer_email() -> str:
+    return "apm-testing-internal-dev@nhs.net"
 
 
 @pytest.fixture()
-def developer_apps(client):
-    return DeveloperAppsAPI(client=client)
+def pds_url() -> str:
+    return f"{BASE_URL}/{PDS_BASE_PATH}"
 
 
 @pytest.fixture()
-def add_asid_to_testapp(developer_apps, nhsd_apim_test_app):
+def add_asid_to_testapp(developer_apps_api,
+                        nhsd_apim_test_app,
+                        developer_email):
```

Comment:
why are we removing logging? I know it doesn't have spineii ecosystem of logs, but isn't some logging useful?

Reply:
See https://github.com/NHSDigital/personal-demographics-service-api/pull/924#discussion_r1361694997